examples/run_generic_hahn_echo_acq.py

```diff
@@ -0,0 +1,128 @@
+# Use echo acq time instead of acq_time_ms
+from pylab import *
+from pyspecdata import *
+from numpy import *
+import SpinCore_pp
+from SpinCore_pp import prog_plen
```

Comment:
here is a third one!! This is not OK.  There should be only one spin echo sequence based on the generic ppg that we can review together.

Reply:
each ppg has one line in the ppg_list changed so I can diff them and see the changes

Reply:
this one changes acq_time_ms to use echo_acq_ms

Reply:
eleted

Reply:
but you never want to do that.